### PR TITLE
Image cropper plugin for media-manager-improvement

### PR DESCRIPTION
**Pull request by euismod2336**
This PR includes the initial plugin which follows the new media manager "way". Currently the plugin is in a proof-of-concept state and more could be added/tweaked.

### CODE REVIEW

**Comment by wilsonge** [comment]
Can we have the robots.txt.dist back please :P

File: plugins/media-editor/imagecropper/imagecropper.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+		if ($input->get('save_copy') === 'on')
+		{
+			$random   = JUserHelper::genRandomPassword(3);
+			$fileName = JFile::makeSafe($random . '_' . $fileName);
+			$fileName = str_replace(' ', '-', $fileName);
+		}
+
+		$result = $image->toFile($filePath . $fileName);
+
+		//todo report result to user
```
Comment (wilsonge) [inline plugins/media-editor/imagecropper/imagecropper.php]: So what's the plan here?

**Comment by jissereitsma** [comment]
@wilsonge lol. @euismod2336 Could you modify the PR so that there are no changes to the original Joomla files, only the addition of the new plugin files? I can see (and @wilsonge noticed) that `robots.txt.dist` is removed, and also `.gitignore` is modified. Those would only be local changes. But great work so far! I tested out the plugin and it works nicely.

File: plugins/media-editor/imagecropper/imagecropper.php (hunk 1 above)
Comment (jissereitsma) [inline plugins/media-editor/imagecropper/imagecropper.php]: @wilsonge Are you referring to the name of the new file?

**Comment by euismod2336** [comment]
@wilsonge the plan is world domination, and this is but the first step...*evil laughter*

`robots.txt.dist` is back and reverted the `.gitignore` :)

**Comment by jissereitsma** [comment]
Looks like solid to me. I hope to provide some updates on the entire project to you and others, but it's great to have this first media plugin to be crossed off the list.

**Pull request merged**